### PR TITLE
create login signup pages

### PR DESCRIPTION
**Pull request by SofieDunt**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/pulses/788451685)

## This PR
Created login and signup pages (and corresponding containers)
- Look consistent with designs
  - Design uses font-family: Circular, used sans-serif as the fallback (in screenshots)
- Functional links to login/signup pages (no forgot password form exists)
  - Added paths to App
- Use AntD 24 Grid System for layout flexibility/consistency

## Screenshots
Left: Design, Right: Finished Page (excluding NavBar)

## Verification Steps
- Compared side-by-side to see if the pages generally matched the design
- Shrunk/maximized window size to check flexibility
- Clicked links to check functionality
- Checked that color codes matched the design

### CODE REVIEW

File: src/containers/signup/Signup.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                                    >
+                                        <Input placeholder="Email"/>
+                                    </Form.Item>
+                                </Col>
+                                <Col span={rSpan}>
+                                    <Form.Item
+                                        name="cemail"
+                                        rules={[{ required: true, message: 'Please confirm your email!' }]}
+                                    >
+                                        <Input placeholder="Confirm Email"/>
```
Comment (florisdobber) [inline src/containers/signup/Signup.tsx:60]: We don't confirm email addresses. That's a mistake in the design sorry, but can you take it out and just make it a long input field like the password ones below?

**Review by florisdobber — CHANGES_REQUESTED** [review]
Looks great overall! Just the minor comment

File: src/containers/signup/Signup.tsx (hunk 1 above)
Comment (SofieDunt) [reply src/containers/signup/Signup.tsx]: Fixed!

**Comment by jackblanc** [comment]
You should be able to fix the CI build with `npm run lint-fix && npm run prettier-fix` :)

**Comment by jackblanc** [comment]
I resolved the merge conflicts from the changes to frontend-scaffold

File: src/containers/login/login.less [hunk 2]
```diff
@@ -0,0 +1,78 @@
+.body-content-container {
```
Comment (jackblanc) [inline src/containers/login/login.less:1]: 💭 Could we share some of these styles between the two containers? It looks like a lot of them are the same between the two pages.

**Comment by jackblanc** [comment]
@SofieDunt I made a bunch of changes to frontend scaffold and wanted to make sure they work properly across downstream repos, so that's the reason for the commits above.

**Review by florisdobber — APPROVED** [review]
Looks good to me!

**Pull request merged**